test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -31,11 +31,26 @@ export function setup() {
 		it('can perform basic code cell functionality', async function () {
 			const app = this.app as Application;
 			await app.workbench.sqlNotebook.newUntitledNotebook();
+			await app.workbench.sqlNotebook.notebookToolbar.waitForKernel('SQL');
 			await app.workbench.sqlNotebook.addCellFromPlaceholder('Code');
 			await app.workbench.sqlNotebook.waitForPlaceholderGone();
 
-			const sampleText: string = 'SELECT * FROM sys.tables';
-			await app.workbench.sqlNotebook.waitForTypeInEditor(sampleText);
+			const text1: string = 'SEL';
```

Comment:
I think it would be best to split these into their own test - each test should ideally be testing a single thing. So one for testing auto-complete and one for testing colorization

Reply:
Yeah I think that makes sense too. I can split it in a future PR when I add more tests for each one.